packages/clients/tanstack-query/src/vue.ts

```
@@ -0,0 +1,401 @@
+    const finalOptions: any = {
+        queryKey,
+        queryFn: ({ queryKey, signal }: any) => {
+            const [_prefix, _model, _op, args] = queryKey;
+            const reqUrl = makeUrl(endpoint, model, operation, args);
+            return fetcher<TQueryFnData, false>(reqUrl, { signal }, fetch, false);
+        },
+        ...options,
+    };
+    return {
+        queryKey,
+        ...useInfiniteQuery(finalOptions),
```

Biome (2.1.2)

[error] This hook is being called from a nested function, but all hooks must be called unconditionally from the top-level component.

This is the call path until the hook.

For React to preserve state between calls, hooks needs to be called unconditionally and always in the same order.
See https://reactjs.org/docs/hooks-rules.html#only-call-hooks-at-the-top-level

(lint/correctness/useHookAtTopLevel)